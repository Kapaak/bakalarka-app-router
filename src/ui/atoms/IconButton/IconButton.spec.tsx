import { ArrowArcLeft } from "@phosphor-icons/react";
import { render } from "@testing-library/react";

import { IconButton } from "./IconButton";

describe("IconButton", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<IconButton icon={<ArrowArcLeft />} />);

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
