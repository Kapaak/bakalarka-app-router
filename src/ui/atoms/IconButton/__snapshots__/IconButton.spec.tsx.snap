// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton should render successfully 1`] = `
<body>
  <div>
    <button
      class="rounded-small disabled:bg-slate-200 disabled:text-slate-400 disabled:border-slate-200 text-black border-black"
    >
      <svg
        fill="currentColor"
        height="1em"
        viewBox="0 0 256 256"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M232,184a8,8,0,0,1-16,0A88,88,0,0,0,65.78,121.78L43.4,144H88a8,8,0,0,1,0,16H24a8,8,0,0,1-8-8V88a8,8,0,0,1,16,0v44.77l22.48-22.33A104,104,0,0,1,232,184Z"
        />
      </svg>
    </button>
  </div>
</body>
`;
